src/Azure.Functions.Cli/Common/StaticSettings.cs

```diff
@@ -1,9 +1,13 @@
-﻿using System;
+﻿using Azure.Functions.Cli.Helpers;
+using System;
 
 namespace Azure.Functions.Cli.Common
 {
     public static class StaticSettings
     {
         public static bool IsDebug => Environment.GetEnvironmentVariable(Constants.CliDebug) == "1";
+
+        public static bool IsTelemetryEnabled => !(EnvironmentHelper.GetEnvironmentVariableAsBool(Constants.TelemetryOptOutVariable)
```

Comment:
shouldn't this just check if that variable is defined? do we need to take its content into account?

Reply:
I figured we want to opt out if it's set to `1` or to `true`. That's what the .NET core CLI does as well (and I have been following their pattern :) ). I can change it to just look for the variable. I think that makes sense to be rather cautious and just assume that if the variable is set, user really wants to opt out.